shuup/admin/static_src/base/js/select.js

```diff
@@ -17,7 +17,7 @@ function activateSelect($select, model, attrs={}) {
         language: "xx",
         minimumInputLength: 3,
         ajax: {
-            url: "/sa/select",
+            url: window.ShuupAdminConfig.browserUrls.select,
```

Comment:
Nice! Can you consider also fixing this https://github.com/Pikkupomo/shuup/blob/acd2c5a1b5ccc7492f94ca40adc38fa2f52c66b6/shuup/admin/static_src/base/js/dropzone.js#L19 same way?